geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -79,7 +78,7 @@ The Redis API for <%=vars.product_name%> currently supports the following comman
 -   **Sets**: SADD, SMEMBERS, SREM
 -   **Strings**: APPEND, DECR, GET, GETRANGE, INCR, INCRBY, INCRBYFLOAT, MGET, SET, SETNX, STRLEN
 
-The following Redis API for <%=vars.product_name%> commands are **unsupported**. Unsupported
+The following <%=vars.product_name%> API for Compatibility with Redis commands are **unsupported**. Unsupported
```

Comment:
I realize we were saying `Redis API for Geode commands` before, but I think it makes sense for this to just be `Redis commands` since we are talking about actual Redis commands that we don't support rather than a special set of commands we've created.